scripts/upgrade-tests.yml

```diff
@@ -0,0 +1,82 @@
+version: '3.5'
```

Comment:
it might be beneficial for future us to move this yml and the test-upgrade-path.sh into a subdirectory so we know the two are related (and should be updated simutanously if needed).